notifier.rst

```diff
@@ -645,11 +673,12 @@ The desktop channel is used to display local desktop notifications on the same
 host machine using :class:`Symfony\\Component\\Notifier\\Texter` classes. Currently,
 Symfony is integrated with the following providers:
 
-===============  ====================================  ==============================================================================
-Provider         Package                               DSN
-===============  ====================================  ==============================================================================
-`JoliNotif`_     ``symfony/joli-notif-notifier``       ``jolinotif://default``
-===============  ====================================  ==============================================================================
+===============  ================================================================
+Provider
+===============  ================================================================
+`JoliNotif`_     **Install**: ``composer require symfony/joli-notif-notifier`` \
+                 **DSN**: ``jolinotif://default``
+===============  ================================================================
```

Comment:
```suggestion
=============  ================================================================
`JoliNotif`_   **Install**: ``composer require symfony/joli-notif-notifier`` \
               **DSN**: ``jolinotif://default``
=============  ================================================================
```

Reply:
I reverted this change because I cannot make it work. Maybe it's some edge-case because this is a single row table.